src/connector/metrics.rs

```diff
@@ -29,6 +29,8 @@ where
     );
 
     histogram!(format!("{}.query.time", tag), start.elapsed());
+    histogram!("query.total_elapsed_time", start.elapsed());
```

Comment:
Question because I really don't know: is it the same metric as the previous line, but with a different name?

Reply:
Yes. It is a more general metric for all queries